### ISSUE TITLE
Error when processing 009-XXX subjects

### ISSUE THREAD

**Issue by sl609** [TeamXenonDuke/xenon-siemens-to-mrd-consortium#9]

**Comment by davidmummy** [comment]
Stealing @sl609's description from our Slack: 

pipeline does not have the procedure to process the data with size 2487 and flip angle  15. (the code expect either 2)
This is not occur in the processing pipeline because we already account for that
        elif raw_fids.shape[0] // 100 == 24:
            [logging.info](http://logging.info/)("Reading in medium dixon data on Siemens Prisma.")
            num_spectra = raw_fids.shape[0] % 100
            data_gas = raw_fids[:-num_spectra][0::2, :]
            data_dis = raw_fids[:-num_spectra][1::2, :]
            n_frames = data_dis.shape[0]
            n_skip_start = 0
            n_skip_end = 0
            grad_delay_x, grad_delay_y, grad_delay_z = -5, -5, -5
Basically we need to add this to the MRD code.
